tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -186,7 +186,7 @@ install_cached_tarball_component() {
 	fi
 
 	local component="${1}"
-	local current_version="${2}"
+	local current_version="${2}-$(git log -1 --pretty=format:"%h" ${repo_root_dir}/tools/packaging/kata-deploy/local-build)"
```

Comment:
Do we want to add this to every cached component, or be more granular in the directories we check for each component (adding this code to the `latest_artefact` of each component).

I'd like to hear @fidencio's opinion on this - especially as he might have directed you to do it this way :)

Reply:
I directed Choi to add this to *every* component, as if something changes here, there's no simple way to know whether we'd need to rebuild the component or not.  So, in order to be on the safe(r) side, and avoid adding this every single where (and possibly miss that in a few places), I've opted for having this as a "global".

@stevenhorsman, do you think the approach is reasonable considering the given rationale?

Reply:
Yep - I think this is ok.